dom0-updates/qubes-dom0-update

```diff
@@ -1,13 +1,12 @@
-#!/bin/bash
-
-find_regex_in_args() {
-    local regex="${1}"
-    shift 1
-
-    for arg in "${@}"; do
-        if echo "${arg}" | grep -q -e "${regex}"; then
-            return 0
-        fi
+#!/bin/bash --
+set -euo pipefail
```

Comment:
Both `set -e` and `set -o pipefail` breaks this in several ways. At the very least, there are several places that operate on `$?` to handle various exit codes (not only errors) - with `set -e` all that will stop working. But then, there is `tar` call that is supposed to pack files on best-effort basis (not all those files may be present on every system) - pipefail breaks this.